### DISCUSSION TITLE
Better Help?

### DISCUSSION THREAD

**Discussion by berndsalewski** [rxhanson/RectanglePro-Community#385] [Ideas]
Hi, 

I am currently evaluating RectPro since I just bought a 27inch monitor and moving & resizing windows is a pain on that dimension. I am quite capable and happy to play around with the tool and discover it myself but cannot help to feel that it's missing some better documentation. There are many things which are not necessarily self-explanatory. Had to figure out on my own what Window throw actually meant. Still not sure what "Rest x Touches, remove all except 1" in the Trackpad settings means. Or improve the in-app tooltips. From a tool I pay for I expect better help. Is there a wiki which I might have missed?

Thanks and keep up the great work.

**Comment by rxhanson** [comment]
Thanks for the feedback. Documentation is a product itself, and with limited time I've favored adding more features to the app over documenting them. Although issues & discussions in GitHub are not the same as software docs, it's been a middle ground that at least provides a searchable interface for targeted usability issues.

With that said, here's a video I made about the window throw that you might find useful:
https://www.youtube.com/watch?v=Vo09H-ln7BQ&t=1s

**Comment by berndsalewski** [comment]
Fair enough :-) thanks for the video.
